### PR TITLE
Chore: speedup country lookup from IP

### PR DESCRIPTION
**Pull request by ifoukarakis**
#### Summary

- [x] Add join bucket
- [x] Compare results

### CODE REVIEW

**Comment by ifoukarakis** [comment]
For reference, this change reduces the query execution time from ~ 250 seconds to 90 seconds.

File: transform/mattermost-analytics/models/staging/geolite/_geolite__models.yml [hunk 1]
```diff
@@ -27,6 +27,25 @@ models:
     columns:
       - name: network
         description: The IPv4 network in CIDR format such as "2.21.92.0/29".
+      - name: cidr_prefix
```
Comment (catalintomai) [inline transform/mattermost-analytics/models/staging/geolite/_geolite__models.yml:30]: based on this: https://docs.digitalocean.com/glossary/cidr/ - sounds like this is the CIDR prefix length.

File: transform/mattermost-analytics/models/staging/geolite/_geolite__models.yml [hunk 2]
```diff
@@ -27,6 +27,25 @@ models:
...
+        description: The CIDR prefix from the address. I.e. for "2.21.92.0/5" it is "5".
+        tests:
+          - dbt_utils.accepted_range:
+              # Accept that the smaller possible value is the one used for creating buckets of ip addresses
+              min_value: 7
+              inclusive: true
+      - name: join_bucket
+        description: |
+          Bucket that the network belongs to. Derived from the network's start IP range, using `/7` as teh mask.
```
Comment (catalintomai) [inline transform/mattermost-analytics/models/staging/geolite/_geolite__models.yml:39]: teh->the

**Review by catalintomai — APPROVED** [review]

File: transform/mattermost-analytics/models/staging/geolite/_geolite__models.yml (hunk 1 above)
Comment (ifoukarakis) [reply transform/mattermost-analytics/models/staging/geolite/_geolite__models.yml:30]: Renaming to `cidr_network_size`

**Pull request merged**